EditorExtensions/Misc/MenuItems/BundleFiles.cs

```diff
@@ -64,8 +64,8 @@ private void BeforeQueryStatus(object sender, params string[] extensions)
 
             _files = ProjectHelpers.GetSelectedFilePaths();
 
-            menuCommand.Enabled = _files.All(file => extensions.Contains(Path.GetExtension(file))) &&
-                                  _files.Count() > 1;
+            menuCommand.Enabled = _files.Any(file => extensions.Contains(Path.GetExtension(file))) &&
+                          _files.Count(file => extensions.Contains(Path.GetExtension(file))) > 1;
```

Comment:
I think all this can be replaced by one line:

```c#
menuCommand.Enabled = _files.Any(file => extensions.Contains(Path.GetExtension(file)));
```

Is it not what we are trying to say here?